paasta_tools/cli/cmds/remote_run.py

```diff
@@ -13,21 +13,183 @@
+        poll_response = client.remote_run.remote_run_poll(
+            args.service,
+            args.instance,
+            start_response.job_name,
+        )
+        if poll_response.status == 200:
+            break
+        time.sleep(3)
+    else:
+        print("Timed out while waiting for job to start")
+        return 1
+
+    if not args.interactive:
+        print("Successfully started remote-run job")
+        return 0
+
+    token_response = client.remote_run.remote_run_token(
+        args.service, args.instance, user
+    )
+
+    exec_command = KUBECTL_CMD_TEMPLATE.format(
+        cluster=args.cluster,
+        namespace=poll_response.namespace,
+        pod=poll_response.pod_name,
+        token=token_response.token,
+    )
+    pty.spawn(shlex.split(exec_command))
+    return 0
+
+
+def paasta_remote_run_stop(
+    args: argparse.Namespace,
+    system_paasta_config: SystemPaastaConfig,
+) -> int:
+    client = get_paasta_oapi_client_with_auth(
+        cluster=get_paasta_oapi_api_clustername(cluster=args.cluster, is_eks=True),
+        system_paasta_config=system_paasta_config,
+    )
+    if not client:
+        print("Cannot get a paasta-api client")
+        return 1
+    response = client.remote_run.remote_run_stop(
+        args.service, args.instance, RemoteRunStop(user=get_username())
+    )
+    print(response.message)
+    return 0 if response.status < 300 else 1
+
+
+def add_common_args_to_parser(parser: argparse.ArgumentParser):
+    service_arg = parser.add_argument(
+        "-s",
+        "--service",
+        help="The name of the service you wish to inspect. Required.",
+        required=True,
+    )
+    service_arg.completer = lazy_choices_completer(list_services)  # type: ignore
+    parser.add_argument(
+        "-i",
+        "--instance",
+        help=(
+            "Simulate a docker run for a particular instance of the "
+            "service, like 'main' or 'canary'. Required."
+        ),
+        required=True,
+    )
+    cluster_arg = parser.add_argument(
+        "-c",
+        "--cluster",
+        help="The name of the cluster you wish to run your task on.",
+        required=True,
+    )
+    cluster_arg.completer = lazy_choices_completer(list_clusters)  # type: ignore
+
+
+def add_subparser(subparsers: argparse._SubParsersAction) -> None:
+    remote_run_parser = subparsers.add_parser(
         "remote-run",
-        help="Schedule adhoc service sandbox on PaaSTA cluster",
-        description=(
-            "`paasta remote-run` is useful for running adhoc commands in "
-            "context of a service's Docker image."
+        help="Run stuff remotely.",
+        description=("'paasta remote-run' runs stuff remotely "),
+    )
+    subparsers = remote_run_parser.add_subparsers(dest="remote_run_command")
+    start_parser = subparsers.add_parser(
+        "start",
+        help="Start or connect to a remote-run job",
+        description=("Starts or connects to a remote-run-job"),
```

Comment:
nit:
```suggestion
        description="Starts or connects to a remote-run-job",
```

Reply:
yaaa, because I had this on a newline, black--------------